### DISCUSSION TITLE
How should I modify it to use it for real-time trading?

### DISCUSSION THREAD

**Discussion by virtualpeer** [edtechre/pybroker#16] [Q&A] [answered]
How should I modify it to use it for real-time trading?

**Comment by edtechre** [comment, marked as answer]
Hi @virtualpeer,

Live trading is currently not supported. However, you can save the models you train with PyBroker and use them in a live trading strategy. 

Please note that the asynchronous nature of live trading makes the needed modifications difficult. If you are still interested, you can begin by looking at the [strategy module](https://github.com/edtechre/pybroker/blob/master/src/pybroker/strategy.py).
